cmd/assign/address/address.go

```diff
@@ -17,13 +16,70 @@ func NewAssignAddressCmd() cli.Command {
 		Usage:     "Assign an address to a specified teamdrive",
 		Action:    CmdAssignAddress,
 		Flags:     []cli.Flag{},
-		UsageText: "<TEAMDRIVE> <ADDRESS>",
+		UsageText: "<TEAMDRIVE-NAME> <ADDRESS> <TYPE> <ROLE>",
 	}
 }
 
+func contains(s []string, e string) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
+
 func CmdAssignAddress(c *cli.Context) {
 
-	client, err := api.CreateClient(App.AppConfig.ServiceAccountFile, App.AppConfig.Impersonate, []string{iam.CloudPlatformScope, admin.AdminDirectoryGroupScope})
+	supportedTypes := []string{
```

Comment:
Can you define them as const please :)

Reply:
See new commit :) Do you know a better way to define them as const? In go you can't make arrays constant :(